code/modules/surgery/organs/tongue.dm

```diff
@@ -87,7 +87,7 @@
 		var/obj/item/organ/tongue/T = H.getorganslot(ORGAN_SLOT_TONGUE)
 		if(!T || T.type != type)
 			continue
-		if(H.dna && H.dna.species.id == "abductor" && user.dna && user.dna.species.id == "abductor")
+		if(H.dna && H.dna.species.id == ROLE_ABDUCTOR && user.dna && user.dna.species.id == ROLE_ABDUCTOR)
```

Comment:
These shoudln't use these. (Species lacking defines is another problem)

Reply:
Seemed like a good idea but I'll change it back. Should I do the same for factions then?

Reply:
Factions are *probably* fine since they're mostly used in the way of antag recognition anyway.